Cargo.toml

```diff
@@ -116,7 +116,7 @@ dashmap = "5.5.3"
 rustc-hash = "1.1.0"
 
 serde = { version = "1.0.204", default-features = false }
-serde_json = { version = "1.0.121", default-features = false }
+serde_json = { version = "1.0.121", default-features = false, features = ["alloc"] }
```

Comment:
I maybe wouldn't include `alloc` in root

Reply:
`serde_json` requires either `std` or `alloc` feature, and `alloc` is subset of `std` so as I understand, `alloc` is required feature